backend/cabin-sorting/types.ts

```diff
@@ -0,0 +1,17 @@
+export type Hacker = {
+    email: string,
+    isAdmin?: boolean,
+    applicationStatus: "Submitted" | "Not Submitted",
```

Comment:
if you move the enums from above down into this file, you can set the type to the corresponding enum instead of hard coding